package.json

```diff
@@ -20,6 +20,7 @@
     "build:main": "cross-env NODE_ENV=production webpack --bail --config webpack.config.main.js",
     "build:renderer": "cross-env NODE_ENV=production webpack --bail --config webpack.config.renderer.js",
     "start": "electron src --disable-dev-mode",
+    "clean":"rm -rf dist/ release/ node_modules/ src/node_modules/ build/ browser/**/*_bundle.js}",
```

Comment:
- missing a white space after `clean:`.
- there is unexpected trailing token `}`.